test/datasets/ljspeech_test.py

```diff
@@ -0,0 +1,80 @@
+import csv
+import os
+
+from torchaudio.datasets import ljspeech
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    normalize_wav,
+    save_wav,
+)
+
+
+class TestLJSpeech(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    data = []
+    transcripts = [
+        "Test transcript 1",
+        "Test transcript 2",
+        "Test transcript 3",
+        (
+            "The examination and testimony of the experts enabled the "
+            "Commission to conclude that 5 shots may have been fired,"
```

Comment:
this might sound nit-picky but I do not find this transcript. In fact, the transcript I found in the actual dataset is same as normalized one.

```
$grep "The examination and testimony of the experts enabled the" LJSpeech-1.1/metadata.csv
LJ001-0031|In 1465 Sweynheim and Pannartz began printing in the monastery of Subiaco near Rome,|In fourteen sixty-five Sweynheim and Pannartz began printing in the monastery of Subiaco near Rome,
```

Since we are kind of using the real sample from the dataset, can we stick with the actual sample?
How about this one? (`1465` -> `fourteen sixty-five`)
```
LJ001-0031|In 1465 Sweynheim and Pannartz began printing in the monastery of Subiaco near Rome,|In fourteen sixty-five Sweynheim and Pannartz began printing in the monastery of Subiaco near Rome,
```

Reply:
Interesting, I took the example off the LJSpeech dataset website, but I didn't cross check it with the actual dataset, so there was probably some inconsistencies. I will use the example you provided. Thanks!